documentation/proc-pages/eng-models/vacuum-vessel.md

```diff
@@ -4,165 +4,178 @@
+2.  The parameters of the 6 arcs are expressed in terms of the overall parameters $\kappa$, $\delta$, A, H and $\theta1$.  (Note these are parameters of the coil or vessel, not the plasma.  Also, $\theta1$ is rather arbitrary, not a meaningful quantity.)
 
 3.  As the analytical formula for the self-inductance is complicated, three surrogate formulae are derived by a regression analysis for the range:  
-1.5 ≤ κ ≤ 2.0,  
+$1.5 ≤ \kappa ≤ 2.0$,  
 1.5 ≤ A ≤ 2.0,   
-0.22 ≤ δ ≤ 0.5,   
-0 < θ1/90◦ ≤ 0.7
+$0.22 ≤ \delta ≤ 0.5$,   
+$0 < \theta1/90^{\circ} ≤ 0.7$ <br>
 H sets the scale and can have any value.
 
-All the inductance formulae are based determining the dimensionless factor ξ in equation 1.  
+All the inductance formulae are based determining the dimensionless factor $\xi$ in the equations below.  
 
 The aspect ratio A of the vacuum vessel will always be smaller than the aspect ratio of the plasma, but it could easily exceed the maximum figure of 2.0 used for the surrogate formula.
 
-Surrogate 1: Equation 2:  
-![image](/uploads/ce2fa0dad1e1358fd6b949c0b3dec8df/image.png)   
-![image](/uploads/8b3e4f684b89d77b72905b5fd3a0f313/image.png)  
-![image](/uploads/bbd3ce64e55634d7012ba8cbf934d6ce/image.png)  
-(Note that ε is not actually part of the formula - it just represents the error.)  
+Surrogate 1: Equation 2[^1]: 
+
+$$\xi = c_0 + c_\kappa \kappa + c_\delta \delta + c_{A1}A + c_{A2}A^2 + c_\theta(\theta_1/90) + \varepsilon,$$
+
+with $c_0 = 4.933$, $c_\kappa = 0.03728$, $c_\delta = 0.06980$, $c_{A1} = −3.551$, $c_{A2} = 0.7629$, and $c_\theta = −0.06298$.
+
+(Note that $\varepsilon$ is not actually part of the formula - it just represents the error.)  
 Maximum error 1.5%.
 
-Surrogate 2 (most accurate): Equation 2 and a correction term Δξ given by   
-![image](/uploads/5dc97896ebb6afb5f114d8320e454a9f/image.png)![image](/uploads/ddf8fb25438418168f420a59762901d4/image.png)  
-![image](/uploads/838e0ddcc0abd57edf80a6e98bc79426/image.png)
+Surrogate 2 (most accurate): Equation 2 and a correction term $\Delta \xi$ given by   
+
+$$\Delta \xi = C_0 + C_\kappa \kappa + C_\delta \delta + C_A A,$$
+
+and had optimum coeﬃcients $C_0 = 0.05808$, $C_\kappa = −0.05846$, $C_\delta = −0.02906$, and $C_A = 0.03008$.
 
-Surrogate 3: ξ is given by
-![image](/uploads/85d704fded3509233791f3cb6c0bd81d/image.png)  
-![image](/uploads/21cb9e7f0d331c9742297e564c698edf/image.png)
-![image](/uploads/d210b3d0b7e2ac76b089fb052059afb5/image.png)  
-maximum error 1.4%.  
-This is the simplest formula and doesn't use θ1, which is useful since θ1 is rather arbitrary.
+Surrogate 3: $\xi$ is given by
+
+$$\xi ' = {C'}_0 + {C'}_\kappa \kappa + {C'}_\delta \delta + {C'}_A A,$$
+
+with ${C'}_0 = 3.026$, ${C'}_\kappa = −0.03238$, ${C'}_\delta = 0.1091$, and ${C'}_A = −1.093$.
+
+Maximum error 1.4%. This is the simplest formula and doesn't use $\theta1$, which is useful since $\theta1$ is rather arbitrary.
```

Comment:
```suggestion
Maximum error 1.4%. This is the simplest formula and doesn't use $\theta_1$, which is useful since $\theta_1$ is rather arbitrary.
```